### ISSUE TITLE
nohlsearch not working

### ISSUE THREAD

**Issue by Happyholic1203** [1995eaton/chromium-vim#311]
After doing a search, setting nohlsearch doesn't cancel the highlighted texts.
I'm using Chrome 43.0.2357.124 (64-bit).
Thanks!

**Comment by zarak** [comment]
Same problem.
Chrome version 45.0.2454.85 (64-bit)

**Comment by zarak** [comment]
Workaround: You can clear the highlighted text by entering a new search expression, then deleting the expression (rather than hitting enter to execute the search).

**Comment by Happyholic1203** [comment]
Yes I understand the workaround, but it'd be better if it's fixed. =P

**Comment by 1995eaton** [comment]
For right now, I've added a workaround: you can map a binding for `clearSearchHighlight`. I've kind of dug myself into a hole with some command bar commands like `:nohlsearch` after I moved the command bar to an iframe, but I'll keep looking for solutions.
